src/routes/resources/+page.svelte

```diff
@@ -30,8 +36,29 @@
     filterObject.push(tagOption)
   }
 
+  function filterBySearchInput(event: CustomEvent<{searchTerm: string}>) {
+    const { searchTerm } = event.detail;
+
+    const options = {
+      includeScore: true,
+      threshold: 0.25,
+      keys: ['description', 'tags', 'title']
```

Comment:
What is the effect of including "tags" in `keys`. Is the intention to also search by tags (e.g. by searching "Activism" get to the resources with that tag?*)

I think limiting the search to the titles and summaries would be good

*to be clear, this usecase work from my testing on this branch